docs/modules/ROOT/pages/how-tos/configuring/prefetching-dependencies.adoc

```diff
@@ -4,33 +4,39 @@ In {ProductName}, you can run a hermetic build by restricting network access to
 
 For every build, Cachi2 generates a software bill of materials (SBOM) where all dependencies are properly declared and pinned to specific versions. Also, Cachi2 ensures that arbitrary code is never executed during the prefetch, meaning, for example, that the build doesn’t pull any undeclared dependencies. Such measures result in very accurate SBOMs and improve the build reproducibility. For more information about SBOMs, see xref:/how-tos/metadata/sboms.adoc[Inspecting SBOMs].
 
-[#available-languages]
-.Available languages
+[#available-package-managers]
+.Available package managers
 
 [cols="1,1"]
 |===
-|**Language**
 |**Package manager**
+|**Programming language**
 
-|xref:gomod[Go]
-|`gomod`
+|xref:gomod[gomod]
+|`Go`
 
-|xref:pip[Python]
-|`pip`
+|xref:pip[pip]
+|`Python`
 
-|xref:npm[Node.js]
-|`npm`
+|xref:npm[npm]
+|`JavaScript`
 
-|xref:rpm[RPM*]
-|`rpm`
+|xref:yarn[yarn]
+|`JavaScript`
 
-|xref:generic[Generic dependencies]
-|`generic fetcher`
+|xref:bundler[bundler]
+|`Ruby`
+
+|xref:rpm*[rpm]
+|`N/A`
+
+|xref:generic[generic]
+|`N/A`
 |===
 
 NOTE: To use Yarn as the package manager, see the link:https://github.com/containerbuildsystem/cachi2/blob/main/docs/usage.md#example-yarn[Cachi2 Usage guide].
```

Comment:
Do we need this note still? If so, does it make sense to put it in the yarn section instead of up here?

Reply:
No, it should be dropped :disappointed: Thanks